### PR TITLE
Add the ability to capture data on the end_line_pattern of non tabula…

### PR DESCRIPTION
**Pull request by ollietulloch**
There is already the ability to capture the data on a start line of a non tabular record using `capture_start_line: true`, this pull request adds the same functionality to end lines.

### CODE REVIEW

**Comment by joshpencheon** [comment]
@ollietulloch is this PR still in progress? Can't recall why it stalled.

**Comment by ollietulloch** [comment]
@joshpencheon I was awaiting any comments for yourself or @timgentry. There were also a few couple of Hound errors that need to be fixed.

File: lib/ndr_import/non_tabular/table.rb [hunk 1]
```diff
@@ -117,6 +117,8 @@ def partition_and_process_non_tabular_lines
             # This is a start line
             start_record(line)
           elsif line =~ @end_line_pattern
+            # Add the end line to the @non_tabular_record (if required) before ending the record
+            @non_tabular_record << line if @capture_end_line
```
Comment (joshpencheon) [inline lib/ndr_import/non_tabular/table.rb]: For symmetry with start_record / @capture_start_line, should this be handled within end_record?

Comment (ollietulloch) [reply lib/ndr_import/non_tabular/table.rb]: Thanks @joshpencheon, this has been done in dc7bf292af3abcff88e0b42da7b703fdd2665b91

**Pull request merged**